src/pages/EntityDetails/EntityDetails.tsx

```diff
@@ -115,21 +116,21 @@ const EntityDetails = () => {
         active: activeView === "apps",
```

Comment:
Was wondering if maybe we could use `ModelTabs` on lines 116, 123,130 and 140 as well?

Reply:
Ah yes, thanks!